modules/facilities_api/app/services/facilities_api/v2/lighthouse/client.rb

```diff
@@ -43,8 +43,6 @@ def get_facilities(params)
             filtered_params.merge!(params.slice(:state))
           elsif params.key?(:zip)
             filtered_params.merge!(params.slice(:zip))
-          elsif params.key?(:facilityIds)
-            filtered_params.merge!(params.slice(:facilityIds))
```

Comment:
Is there no more facilityIds parameter? I don't know if we used it in v0, but just wondering what happened?

Reply:
There is, facilityIds is used in v1, but it was covered already on line 36 of this file. Had a little duplication there 🙃 

Just for reference, v0 of LH Facilities API uses `ids`, while v1 uses `facilityIds`.

Reply:
Got it. Thanks